cmd/rookflex/cmd/root.go

```diff
@@ -79,7 +79,9 @@ func log(client *rpc.Client, message string, isError bool) {
 		Message: message,
 		IsError: isError,
 	}
-	client.Call("Controller.Log", log, nil)
+	if err := client.Call("Controller.Log", log, nil); err != nil {
+		panic(err)
```

Comment:
In this case let's actually ignore the error. We can't log an error since the attempt to log failed, and we shouldn't panic because we want the original errors to be returned if there was another failure. So let's just add a comment here.